Source/RapyutaSimulationPlugins/Private/Robots/RRBaseRobot.cpp

```diff
@@ -63,7 +64,7 @@ void ARRBaseRobot::PostInitializeComponents()
     }

```

Comment:
Need to change the log text few lines above which still uses `PostInitializeComponents`